tests/graph/test_sepal.py

```diff
@@ -0,0 +1,54 @@
+from anndata import AnnData
+
+from pandas.testing import assert_frame_equal
+import numpy as np
+
+from squidpy.gr import sepal, spatial_neighbors
+
+UNS_KEY = "sepal_score"
+
+
+def test_sepal_seq_par(adata: AnnData):
+    """Check whether sepal results are the same for seq. and parallel computation."""
+    spatial_neighbors(adata, coord_type="visium")
+    adata.var["highly_variable"] = np.random.choice([True, False], size=adata.var_names.shape, p=[0.005, 0.995])
+
+    sepal(adata, max_nbrs=6)
```

Comment:
Could you please parametrize the test with max. neighs = [4, 6]?
Same for the below test
Also both tests are flaky because of the `np.random.choice` without seed (could lead to CI randomly breaking), please add seed.

Reply:
mmh problem is that i would have to parameterize with two dataset (fixtures), can't use the same dataset for both hex and square. I tried a bit but didn't manage

Reply:
Ok, I'd say we can leave as it is, since 1 test use 4, the other uses 6.